.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure git
```

Comment:
Should we add some kind of comment explaining why this is necessary?

Reply:
I think this is only needed if we try to commit the changes that the hook stages- we can just remove it.